src/CasProcessor.php

```diff
@@ -87,7 +87,12 @@ private static function parseAttributes(\DOMElement $node): array {
       else {
         $value = $child->nodeValue;
       }
-      $attributes[$name] = $value;
+      if ($name === 'group') {
```

Comment:
Can't we figure out if the attribute is an array without hardcoding the name of it?  Also, isn't the `group` attribute one array level down?

Also, no tests for this?